src/webchat-ui/components/presentational/HomeScreen.tsx

```diff
@@ -141,31 +144,11 @@ export const HomeScreen: React.FC<IHomeScreenProps> = props => {
 		onStartConversation,
 	} = props;
 
-	// TODO: Load buttons from new endpoint config
-	const buttons: IWebchatButton[] = [
-		{
-			title: "Cognigy Homepage",
-			type: "web_url",
```

Comment:
According to the spec, the buttons will be either of type postback or 'url'. ActionButton component handles only links of type 'web_url'. So, when I add a conversation starter url according to the spec, it does not show up on the UI. We need to do a type conversion here or handle url type in chat-components

Reply:
This is same in Chat options screen and in the teaser message